backend/src/db/migrations/20230823060705_create_surveys_table.ts

```diff
@@ -7,6 +7,11 @@ const ColumnName = {
   ID: 'id',
```

Comment:
This migration won't run on production. Never change old migrations, we already talked about this when the project started. Create a separate migration to add new columns